CHANGELOG.md

```diff
@@ -1,4 +1,38 @@
 # Changelog
+## [1.3.0] - 2021-03-24
+### Changes
+- Use inclusive language
+Use inclusive language in docs, variable names, function names and parameter names
+  - "slave" is deprecated in favor of "port"
+  - "master" is deprecated in favor of "controller"
+
+### New features
+- Support ipv6.method disabled
+- Support dns_options for IPv4 config
+
+  The list of supported DNS options is:
```

Comment:
This is too verbose for the changelog

Reply:
How about now?

Reply:
Please remove the list of options from here.

Reply:
The list of DNS options was removed.